core/src/test/scala/org/apache/spark/scheduler/FakeTask.scala

```diff
@@ -58,4 +57,18 @@ object FakeTask {
     }
     new TaskSet(tasks, stageId, stageAttemptId, priority = 0, null)
   }
+
+  def createShuffleMapTaskSet(numTasks: Int, stageId: Int, stageAttemptId: Int,
+    prefLocs: Seq[TaskLocation]*): TaskSet = {
```

Comment:
nit: multi-line method definitions should have each param on its own line, indented 4 spaces

```scala
def createShuffleMapTaskSet(
    numTasks: Int,
    stageId: Int,
    stageAttemptId: Int,
    prefLocs: Seq[TaskLocation]*): TaskSet = {
```